### PR TITLE
js documentation added in convertimagetobyte

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/includes/ConvertImageToByte.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
 const reader = new FileReader();
 const fileByteArray = [];
 
-function getConvertedImage(data) {
+/**
+ * Converts the image file to byte array.
+ * @param {file} image file.
```
Comment (arrogant-ant) [inline src/includes/ConvertImageToByte.js:6]: does it support image files with only some specific extensions like .jpg?

File: src/includes/ConvertImageToByte.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+ * @return {Array} byte array consist of zeroes and ones.
```
Comment (arrogant-ant) [inline src/includes/ConvertImageToByte.js:7]: `consist of zeroes and ones`  is not needed as this is comment.

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/includes/ConvertImageToByte.js (hunk 2 above)
Comment (alias01) [reply src/includes/ConvertImageToByte.js:7]: done

File: src/includes/ConvertImageToByte.js (hunk 1 above)
Comment (alias01) [reply src/includes/ConvertImageToByte.js:6]: yes it supports

**Pull request merged**